### ISSUE TITLE
docs: Add list of pylhe citations

### ISSUE THREAD

**Issue by matthewfeickert** [scikit-hep/pylhe#181]
We should get docs up in general, but @lukasheinrich pointed out that we should probably be tracking pylhe citations as well.

* [Probing compressed higgsinos with forward protons at the LHC](https://inspirehep.net/literature/2140007) (2023)

**Comment by matthewfeickert** [comment]
Yup, this is part of the reason I mentioned it originally, to make sure that it gets put on the Scikit-HEP page for sure. I'll open a PR later this week to get all of these that are missing up there.

I think there are some advantages to having projects listing their own citations in their docs as they can go into different formats then what we have on the Scikit-HEP page (which is a format that I like for a high level ecosystem summary)...but that also would require me to knock out Issue #30 before that's a relevant issue here. :P

**Comment by eduardo-rodrigues** [comment]
OK, fair enough. Advance thanks 👍.

**Comment by matthewfeickert** [comment]
@lukasheinrich pointed out [On the BSM reach of four top production at the LHC](https://inspirehep.net/literature/2633019) by Anisha, Oliver Atkinson, Akanksha Bhardwaj, Christoph Englert, Wrishik Naskar, Panagiotis Stylianou today as well. :+1:

**Comment by eduardo-rodrigues** [comment]
Indeed! The list in Zenodo is far from complete! I've been trying hard to update regularly the list of publications using any of Scikit-HEP at the page https://scikit-hep.org/projusers/publications/. Help welcome - do not hesitate to prepare a PR if you bump into something missing.

BTW, you will see that pylhe is in fact one of our most cited packages (especially given its size). It would be great to prepare at some point a JOSS paper, as that will be picked up by inspirehep.

**Comment by APN-Pucky** [comment]
> It would be great to prepare at some point a JOSS paper, as that will be picked up by inspirehep.

That sounds good. It could be combined with a 1.0.0 release. Maybe we should collect a list of TODOs/tag-issues before 1.0.0 release? First thing that comes to my mind is some sphinx doc webpage, also we probably want to test/fix #213.

**Comment by eduardo-rodrigues** [comment]
Good idea! Sounds like a target for the quieter days of Summer ... I add this to my list of things to check/follow.
